lib/Dialog/Dialog.js

```diff
@@ -153,6 +165,12 @@ module.exports = kind(
 	*/
 	create: function () {
 		Popup.prototype.create.apply(this, arguments);
+		if(this.bodyComponents) {
+			this.$.message.destroy();
```

Comment:
hi @aarontam Yes, it works. I committed the new code as you suggested. Please review it.
Thanks